ViAn/Analysis/analysismethod.cpp

```diff
@@ -159,16 +162,17 @@ void AnalysisMethod::run() {
  * Checks if path exists and if it does adds a number to the end to prevent
  * that path overwrites the old one
  */
-std::string AnalysisMethod::check_save_path(std::string path, int increment) {
-    std::string new_path = path + std::to_string(increment);
+std::string AnalysisMethod::check_save_path(std::string name, int increment) {
```

Comment:
Is there a better way to do this? Beacuse this function is not the best. Also how does what you're working on affect this since the temporary map will be removed?